TerminalDocs/distributions.md

```diff
@@ -96,6 +96,13 @@ If you wish to reenable portable mode, you can create a new `.portable` marker f
 You can upgrade a portable mode installation of Windows Terminal by moving the `.portable` marker file and the
 `settings` directory to a newly-extracted unpackaged version of Windows Terminal.
 
+### Portable mode FAQs
+
+#### Why don't ms-appdata URLs work in Portable mode?
+Prior to portable mode, a common heuristic to reference images in `settings.json` would be to use `ms-appdata:///Local`. 
```

Comment:
`heuristic` feels like a very strange word to use here. This is more of a "practice".